README.md

```diff
@@ -49,8 +49,9 @@ Please follow the [Toolchain instructions](https://github.com/IBM/container-jour
 
 ## Part B: Explore Istio resiliency features: Circuit Breakers and Fault Injection
 
-3. [Add resiliency feature - Circuit Breakers](#3-add-resiliency-feature---circuit-breakers)
-4. [Add resiliency feature - Timeouts and Retries](#4-add-resiliency-feature---timeouts-and-retries)
+3. [Add resiliency feature - Circuit Breakers (Maximum connections and pending requests)](#3-add-resiliency-feature---circuit-breakers-maximum-connections-and-pending-requests)
+4. [Add resiliency feature - Circuit Breakers (Load balancing pool ejection)](#4-add-resiliency-feature---circuit-breakers-load-balancing-pool-ejection)
```

Comment:
I will remove this Add resiliency feature in the beginnning - Maybe you can have bold heading at top before we get to step 3, 4 5 which says Add resiliency features